genestack_client/metainfo_scalar_values.py

```diff
@@ -39,22 +39,37 @@ def _xstr(arg):
 class StringValue(MetainfoScalarValue):
     _TYPE = 'string'
 
+    def get_string(self):
+        return self._xstr(self.get('value'))
+
 
 class BooleanValue(MetainfoScalarValue):
     _TYPE = 'boolean'
 
+    def get_boolean(self):
+        return bool(self.get('value'))
+
 
 class IntegerValue(MetainfoScalarValue):
     _TYPE = 'integer'
 
+    def get_integer(self):
```

Comment:
May be use properties like in task library?  And for single one we always use `value`